app/views.py

```diff
@@ -81,11 +81,33 @@ def progressreport():
     """Renders the report page."""
     return render_template('report.html', title=' | Progress Report')
 
-@app.route('/create', methods=['GET', 'POST'])
-def create():
-    """Renders the create page for educators."""
-    form = CreateQnForm()
-    return render_template('create.html', title=' | Create', form=form)
+@app.route('/createclass', methods=['GET', 'POST'])
+def createclass():
+    """Renders the create class page for educators."""
+    form = CreateClass()
+    if form.validate_on_submit():
+        Group.name = form.className.data
+        db.session.commit()
+        return redirect(url_for('createclasssuccess'))
+    return render_template('createclass.html', title=' | Create Class', form=form)
+
+@app.route('/createclasssuccess', methods=['GET'])
+def createclasssuccess():
```

Comment:
Wouldn't you want to immediately redirect to the class then? What's the use of this page?

Reply:
This page has a routes to creating another class, entering the class or going back to their dashboard

Reply:
Does the page display all of their classes? Or just the newly created class.

Reply:
Just the newly created class. All the classes can be accessed through their dashboard